File: itJob/WeChat/models/applicant.js
import { HTTP } from '../utils/http.js'
class Applicant extends HTTP {
  getApplicantByUserId(id,sCallback) {
    this.request({
      url: '/applicant/getOneByUserId',
      method: 'get',
      data: {
        userid:id,
      },
      success: (res) => {
        sCallback(res)
      }
    })
  }
  updateApplicant(person, sCallback){
    this.request({
      url: '/applicant/updateApplicant',
      method: 'post',
      data: {
        id:person.id,
        userId:person.userid,
        name:person.name,
        sex:person.sex,
        age:person.age,
        icon:person.icon,
        education:person.education,
        phone:person.phone,
        city:person.city,
        address:person.address,
        email:person.email,
        introduction:person.introduction,
      },
      success: (res) => {
        sCallback(res)
      }
    })
  }
}
export { Applicant }

File: itJob/src/main/java/com/itJob/bean/Applicant.java
package com.itJob.bean;

import com.baomidou.mybatisplus.annotations.TableField;
import com.baomidou.mybatisplus.annotations.TableId;
import com.baomidou.mybatisplus.annotations.TableName;
import com.baomidou.mybatisplus.enums.IdType;
import lombok.Data;

/**
 * @Description: 求职者 类
 * @Author: LRJ
 * @Date: 2019/12/27 16:09
 */
@Data
@TableName("t_applicant")
public class Applicant {
    @TableId(value = "applicant_id",type = IdType.UUID)
    String id;
    String userId;
    @TableField(value ="applicant_name")
    String name;
    String sex;
    Integer age;
    String icon;
    String education;
    String phone;
    String city;
    String address;
    String email;
    String introduction;
}
